Contentless/Contentless.nuspec

```diff
@@ -10,7 +10,7 @@
         <repository type="git" url="https://github.com/Ellpeck/Contentless" />
         <readme>README.md</readme>
         <icon>Logo.png</icon>
-        <version>3.0.7</version>
+        <version>3.0.8</version>
```

Comment:
Please don't increase the version as part of this pull request, the current workflow is to increase the version separately as its own commit (I didn't use to stick to this in the past so sorry for any confusion about how this was done in the past in this repo!)

Reply:
Fixed